setup.cfg

```diff
@@ -50,4 +59,4 @@ addopts =
     --cov-config .coveragerc --cov=./ --cov-report=term
 
 filterwarnings =
-    ignore::django.utils.deprecation.RemovedInDjango20Warning
```

Comment:
We will see all 2.0 warning now during tests ... and that is a good thing.

Reply:
And shouldn't be for long, either :)